### ISSUE TITLE
Not working with with node-mapnik >=3.4.10, mapnik v3.0.9

### ISSUE THREAD

**Issue by eschwartz** [mapnik/mapnik-reference#112]
Running mapnik-reference when using node-mapnik v3.4.10 or higher causes an error:

```
Unknown mapnik-reference version: '3.0.9'
    at Object.module.exports.load ([...]/mapnik-reference/index.js:25:8)
```

node-mapnik v3.4.10 started using mapnik v3.0.9

**Comment by StevenLooman** [comment]
It seems that 8.5.2 (the current version in the repository, containing support for version 3.0.9) was not published (retracted maybe?) The fix would be to publish it again.

@flippmoke Can you publish this version (again)?

**Comment by flippmoke** [comment]
Published `8.5.2`

**Comment by StevenLooman** [comment]
Thank you, this fixed it for me.
